### DISCUSSION TITLE
Upgrading Renault Scenic to Carplay issues

### DISCUSSION THREAD

**Discussion by brandonongsc** [cedricp/ddt4all#1058] [Q&A]
Hello!,

i am trying to upgrade to carplay but faced many unknown issues. i am using a Wifi version of OBD2 which i am able to connect to successfully 

so here are the issues:

1. using the latest release of ddt4all here, i am unable to even connect to the wifi OBD2 
![unable connect port2](https://github.com/cedricp/ddt4all/assets/41996688/e365031b-35cb-4b4b-afb1-583f9ea9917f)

2. using an older version of ddt4all, i don't have the port connecting issue like 1.. i followed some guides i found on the web to navigate to the RadNav 2.0 configuration to update SPVR (to present), Carplay (to authorised) when i click the yellow button to 'Write' configuration, 1 of the field 'SED Level 1 min' complained of some NO DATA error. 
![radnav](https://github.com/cedricp/ddt4all/assets/41996688/81ec2b7b-95dc-4562-875c-e902c6c41af5)

something i noted which is when i click the 'READ ALL' button, it doesn't seem to have any changes also, and it gives me the impression i am probably still not connect to the car's ECU (just my impression, am not sure)

3. there is this option to do CAN SCAN which i saw on the web where it is able to load some MDF thing.. for my case, it loads for about 20seconds progressively to 100%, then nothing comes up. which i don't quite understand also

any gurus here have any insights to share how i can overcome the problems?

**Comment by KarelSvo** [comment]
log files
